seqr/views/apis/summary_data_api.py

```diff
@@ -87,8 +87,11 @@ def saved_variants_page(request, tag):
     if tag == 'ALL':
         saved_variant_models = SavedVariant.objects.exclude(varianttag=None)
     else:
-        tag_type = VariantTagType.objects.get(name=tag, project__isnull=True)
-        saved_variant_models = SavedVariant.objects.filter(varianttag__variant_tag_type=tag_type)
+        tags = tag.split(';')
+        saved_variant_models = SavedVariant.objects.filter(
+            varianttag__variant_tag_type__name__in=tags,
+            varianttag__variant_tag_type__project__isnull=True,
```

Comment:
It returns more saved variants than we need. But the front end will filter out the proper variants to display.

Reply:
we should be doing all the filtering on the backend for the summary data page. Why not keep the 2 distinct queries we had before so that we actually only retunr the correct variants:
```
tag_types = VariantTagType.objects.filter(name__in=tags, project__isnull=True)
saved_variant_models = SavedVariant.objects.filter(varianttag__variant_tag_type__in=tag_types)
```

Reply:
> tag_types = VariantTagType.objects.filter(name__in=tags, project__isnull=True)
> saved_variant_models = SavedVariant.objects.filter(varianttag__variant_tag_type__in=tag_types)

I did some tests and found the result is the same. I can't find a Django filter that means our requirements. What I could do is add a Python filter afterward.

Reply:
It works. Can it be a concern about having many DB queries?

Reply:
django converts that to 1 db query. Chaining `filter` statements does not execute a query